controllers/metrics.go

```diff
@@ -16,6 +16,8 @@ const (
 const (
 	LastSyncTimestampSeconds    = "last_sync_timestamp_seconds"
 	DRPolicySyncIntervalSeconds = "policy_schedule_interval_seconds"
+	LastSyncDurationSeconds     = "last_sync_duration_seconds"
+	LastSyncDataBytes           = "last_sync_data_bytes"
```

Comment:
Maybe sort the constants so related constants (LastSync...) are grouped together?

Reply:
done